common/views/components/styled/AlignFont.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, FunctionComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import TogglesContext from '@weco/common/views/components/TogglesContext/TogglesContext';
+
+type Props = {
+  className?: string;
+  children: ReactNode;
+};
+
+const Align = styled.span<{ isOn: boolean }>`
+  .fonts-loaded & {
+    ${props =>
+      props.isOn &&
+      `
+      transform: translateY(-${props.theme.fontVerticalOffset});
+      display: inline-flex;
```

Comment:
why do we need inline-flex?

Reply:
It has to be something other than `inline` for the `translateY` to have any effect. Could be `inline-block`, maybe?